tests/test_graft.py

```diff
@@ -95,8 +96,8 @@ def test_graft_axon_on_synthesized_cell():
     graft.graft_axon(synthesized_cell, donor_neuron)
     axon = graft.find_axon(synthesized_cell)
     assert_array_almost_equal(axon.points,
-                              [[5.110419, 5.486378, 4.9647303],
-                               [5.110419, 1.486378, 4.9647303]])
+                              [[5.1272364, 5.4825425, 4.9689593],
```

Comment:
I wonder why those changes? Is it due to new version of numpy? In that case should its version be fixed in setup.py?

Reply:
Actually, this change is expected and made me discover that we were not using `np.random.uniform` correctly.
`np.random.uniform(low=0, high=1)` automatically sorts the `low` and `high` arguments, so when calling `np.random.uniform(2 * np.pi)`, we were actually calling `np.random.uniform(low=1, high=2 * np.pi)`... This behavior is very strange but they did not want to fix it for `np.random.uniform` because it is frozen and should no more be used, they only fixed it for `np.random.Generator.uniform`: https://github.com/numpy/numpy/pull/17921

Reply:
Great, thank you for explanation. Up to you to merge.

Reply:
Wow, nice catch.

Reply:
Thanks :)